schemas/serverConfig.json

```diff
@@ -46,7 +46,7 @@
         },
         "serverPort": {
             "description": "The TCP port number for the server.",
-            "type": "string"
+            "type": "integer"
```

Comment:
This is going to break production, so we should be ready for it.

Reply:
Right. Is this change worth making? I don't think we can really have a non-integer port, but maybe there's a reason this was a string to start with?

Reply:
I can't remember, it's been this way for so long :-) I believe that `port` must be an integer, but we also don't really rely on it in any way -- we just concat the strings. It might be easier to just leave it as is.